Source/PeleLMeX_Rad.cpp

```diff
@@ -0,0 +1,88 @@
+#include <PeleLMeX.H>
+#include <PeleLMeX_Derive.H>
+
+#include <PeleLMRad.hpp>
+
+void
+PeleLM::RadInit()
+{
+  amrex::Print() << "Radiation model is activated "
+                 << "\n";
```

Comment:
just use `std::endl`

Reply:
Notes
Use of std::endl in place of '\n', encouraged by some sources, may significantly degrade output performance.
from
https://en.cppreference.com/w/cpp/io/manip/endl

But if you insist, there is no problem to change it.

Reply:
Well that's a definite TIL for me! Thank you. It probably has no impact here because we don't actually do much IO but I can see where it might matter. For us, this would be a style decision. But I don't care that much